decidim-conferences/app/commands/decidim/conferences/admin/destroy_conference_admin.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Conferences
+    module Admin
+      # A command with all the business logic when destroying an conference
+      # admin in the system.
+      class DestroyConferenceAdmin < Rectify::Command
+        # Public: Initializes the command.
+        #
+        # role - the ConferenceUserRole to destroy
+        # current_user - the user performing this action
+        def initialize(role, current_user)
+          @role = role
+          @current_user = current_user
+        end
+
+        # Executes the command. Broadcasts these events:
+        #
+        # - :ok when everything is valid.
+        # - :invalid if the form wasn't valid and we couldn't proceed.
+        #
+        # Returns nothing.
+        def call
+          destroy_role!
+          broadcast(:ok)
+        end
+
+        private
+
+        attr_reader :role, :current_user
+
+        def destroy_role!
+          extra_info = {
+            resource: {
+              title: role.user.name
```

Comment:
Add the id of the destroyed role?

Reply:
In any admin_log we add the id of the element....
This is a decidiscion that @decidim/lot-core and @decidim/product should take